test/services/cache/ExportsItemBuilder.scala

```diff
@@ -103,7 +102,12 @@ trait ExportsItemBuilder {
     cache =>
       cache.copy(
         packageInformation = Some(
-          cache.packageInformation.getOrElse(List.empty) :+ PackageInformation(Some(typesOfPackages), Some(numberOfPackages), Some(shippingMarks))
+          cache.packageInformation.getOrElse(List.empty) :+ PackageInformation(
+            UUID.randomUUID().toString,
```

Comment:
Nit: And here it's inconsistent. I am aware that we can put there anything, but if you consider tests as some kind of a documentation of how should the code work, then consistency here is recommended in my opinion.

Reply:
Nonsense.  You test to the API not the implementation.